src/material-design-lite.scss

```diff
@@ -48,3 +48,4 @@
 @import "tooltip/tooltip";
 @import "shadow/shadow";
 @import "grid/grid";
+@import "selectfield/selectfield";
```

Comment:
Should be alphabetically placed in the list. Shadow and Grid should always be  last.